### PR TITLE
DOCSUP-14458: Edit and translate to Russian (two settings and subBitmap function)

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #26313 and #27234

### CODE REVIEW

File: docs/ru/operations/settings/settings.md [hunk 1]
```diff
@@ -801,12 +801,32 @@ ClickHouse может парсить только базовый формат `Y
...
-Возможное значение:
+Возможные значения:
 
 -   Положительное целое число.
 
 Значение по умолчанию: 2013265920.
 
+## merge_tree_clear_old_temporary_directories_interval_seconds {#setting-merge-tree-clear-old-temporary-directories-interval-seconds}
+
+Задает интервал в секундах в ClickHouse для выполнения очистки старых временных каталогов.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:812]:
```suggestion
Задает интервал в секундах для удаления старых временных каталогов на сервере ClickHouse.
```

File: docs/ru/operations/settings/settings.md [hunk 2]
```diff
@@ -801,12 +801,32 @@ ClickHouse может парсить только базовый формат `Y
...
+
+Возможные значения:
+
+-   Положительное целое число.
+
+Значение по умолчанию: `60` секунд.
+
+## merge_tree_clear_old_parts_interval_seconds {#setting-merge-tree-clear-old-parts-interval-seconds}
+
+Задает интервал в секундах в ClickHouse для выполнения очистки старых кусков, WAL-ов и мутаций.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:822]:
```suggestion
Задает интервал в секундах для удаления старых кусков данных, журналов предзаписи (WAL) и мутаций на сервере ClickHouse .
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 3]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
...
+**Синтаксис**
+
+``` sql
+subBitmap(bitmap, offset, cardinality_limit)
+```
+
+**Аргументы**
+
+-   `bitmap` – битмап. Тип: [Bitmap object](#bitmap_functions-bitmapbuild).
+-   `offset` – количество смещений. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:107]:
```suggestion
-   `offset` – смещение. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 4]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
...
+-   `cardinality_limit` – верхний предел подмножества. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
+
+**Возвращаемое значение**
+
+Подмножество битмапа.
+
+Тип: `Bitmap object`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:114]:
```suggestion
Тип: [Bitmap object](#bitmap_functions-bitmapbuild).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md (hunk 3 above)
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:107]:
```suggestion
-   `offset` – смещение. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 5]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
...
+
+``` sql
+subBitmap(bitmap, offset, cardinality_limit)
+```
+
+**Аргументы**
+
+-   `bitmap` – битмап. Тип: [Bitmap object](#bitmap_functions-bitmapbuild).
+-   `offset` – количество смещений. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
+-   `cardinality_limit` – верхний предел подмножества. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:108]:
```suggestion
-   `cardinality_limit` – максимальное число элементов возвращаемого подмножества. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 6]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
 └───────────────────────────┘
 ```
 
+## subBitmap {#subBitmap}
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:94]:
```suggestion
## subBitmap {#subbitmap}
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 7]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
...
+
+Создает подмножество битмапа, ограниченное параметром `cardinality_limit` со смещением `offset`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:96]:
```suggestion
Возвращает элементы битмапа, расположенные между `offset` и `cardinality_limit`, по аналогии со строковой функцией [substring](string-functions.md#substring).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md [hunk 8]
```diff
@@ -92,6 +91,44 @@ SELECT bitmapToArray(bitmapSubsetLimit(bitmapBuild([0,1,2,3,4,5,6,7,8,9,10,11,12
...
+**Синтаксис**
+
+``` sql
+subBitmap(bitmap, offset, cardinality_limit)
+```
+
+**Аргументы**
+
+-   `bitmap` – битмап. Тип: [Bitmap object](#bitmap_functions-bitmapbuild).
+-   `offset` – смещение. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:107]:
```suggestion
-   `offset` – смещение первого из выбираемых элементов. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/en/sql-reference/functions/bitmap-functions.md [hunk 9]
```diff
@@ -137,15 +137,15 @@ subBitmap(bitmap, offset, cardinality_limit)
 
 **Arguments**
 
--   `bitmap` – [Bitmap object](#bitmap_functions-bitmapbuild).
--   `offset` – the number of offsets. Type: [UInt32](../../sql-reference/data-types/int-uint.md).
+-   `bitmap` – The bitmap. Type: [Bitmap object](#bitmap_functions-bitmapbuild).
+-   `offset` – The value of the offset. Type: [UInt32](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/bitmap-functions.md:141]:
```suggestion
-   `offset` – The position of the first element. Type: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/ru/sql-reference/functions/bitmap-functions.md (hunk 8 above)
Comment (olgarev) [inline docs/ru/sql-reference/functions/bitmap-functions.md:107]:
```suggestion
-   `offset` – позиция первого элемента возвращаемого подмножества. Тип: [UInt32](../../sql-reference/data-types/int-uint.md).
```

File: docs/en/sql-reference/functions/bitmap-functions.md [hunk 10]
```diff
@@ -137,15 +137,15 @@ subBitmap(bitmap, offset, cardinality_limit)
...
 -   `cardinality_limit` – The subset cardinality upper limit. Type: [UInt32](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/bitmap-functions.md:142]:
```suggestion
-   `cardinality_limit` – The maximum number of elements in the subset. Type: [UInt32](../../sql-reference/data-types/int-uint.md).
```

**Comment by sevirov** [comment]
@FArthur-cmd Please, review this PR.

**Review by FArthur-cmd — APPROVED** [review]

**Comment by FArthur-cmd** [comment]
@Mergifyio update

**Pull request merged**